tests/test_connection.py

```diff
@@ -1,58 +1,19 @@
-import mongoengine
-import mongomock
+from mongoengine.context_managers import switch_db
 import pymongo
 from pymongo.errors import InvalidURI
+from pymongo.read_preferences import ReadPreference
 
-from flask_mongoengine import InvalidSettingsError, MongoEngine
-from tests import FlaskMongoEngineTestCase
+from flask_mongoengine import MongoEngine
 
+from .tests import FlaskMongoEngineTestCase
```

Comment:
Looks like this breaks the tests.

https://travis-ci.org/MongoEngine/flask-mongoengine/jobs/178375073#L529